public/docs/ts/latest/guide/architecture.jade

```diff
@@ -73,18 +73,19 @@ block modules-are-optional
        Find setup and organization clues in the JavaScript track (select it from the combo-box at the top of this page)
        which demonstrates Angular 2 development with plain old JavaScript and no module system.
 
+- var _app_comp_filename = _docsFor == 'dart' ? 'app_component.dart' : 'app.component.ts';
 :marked
   Perhaps the first module we meet is a module that exports a *component* class.
   The component is one of the basic Angular blocks, we write a lot of them,
   and we'll talk about components in the next segment. For the moment it is enough to know that a
   component class is the kind of thing we'd export from a module.
 
-  Most applications have an `AppComponent`. By convention, we'll find it in a file named `app.component.ts`.
+  Most applications have an `AppComponent`. By convention, we'll find it in a file named `!{_app_comp_filename}`.
   Look inside such a file and we'll see a declaration like this one.
```

Comment:
Replace like with such as:
Look inside such a file and we'll see a declaration such as this one.